astropy/modeling/fitting.py

```diff
@@ -1348,9 +1350,10 @@ def _fitter_to_model_params(model, fps):
             if model.tied[name]:
                 value = model.tied[name](model)
                 slice_ = param_metrics[name]['slice']
-                model.parameters[slice_] = value
+
                 # To handle multiple tied constraints, model parameters
                 # need to be updated after each iterration.
+                parameters[slice_] = value
```

Comment:
Is this fix *in addition* to the fix that is done in #9601 ? Or does this PR just need a rebase?

Reply:
It's a correction to the one in #9601. I did not mean to leave the update to `model.parameters` in that PR and I noticed the error when rebasing this one.